Labs/Lab03/calculator.py

```diff
@@ -0,0 +1,94 @@
+
+import math
+# Lab 3
+
+available_functions =  ['multiply', 'divide', 'add', 'subtract']
+
+def check_allowed_function(operation):
+    value = operation in available_functions
+    return (value)
+
+def get_inputs():
+    operation = input("What is the operation? (enter q to quit)\n>>> ")
+    operation = operation.lower()
+    if(operation == 'q'):
+        exit()
+
+    if(not check_allowed_function(operation) ):
+        raise NameError(f'Currently this calculator only supports {available_functions}')
+
+    first_number = float(input("What is the first number? : "))
+    second_number = float(input("What is the second number? : "))
+    
+    return (operation, first_number, second_number)
+
+
+
+# Operations:
+#   - add : +
+#   - subtract : -
+#   - multiply : *
+#   - divide : /
+
+def add(x, y):
+    try:
+        result = x + y
+    except:
+        raise ValueError
```

Comment:
Are you sure this will handle cases where the user enter a letter instead of a digit? Or would you simply be replacing a TypeError with a Value Error? Consider failing in a more graceful way. But otherwise, you did an awesome job!!!